### PR TITLE
docs: running tracee docker container on amd64 and arm64

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Regarding the issue detailed in #3454

### CODE REVIEW

File: docs/getting-started/docker-quickstart.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+This section details how you can run Tracee through a container image.
+
+## Prerequisites
+
+!!! The container image currently does not support AArch64 architecture. Please ensure that you are on x64 system.
+
+Additionally, please ensure that Docker or another container runtime is working on your machine. 
+
+Note that we will provide support for multiple platforms, such as AArch64, in upcoming releases.
+In the meantime, have a look at [the Kubernetes guide](./kubernetes-quickstart).
```
Comment (yanivagman) [inline docs/getting-started/docker-quickstart.md:12]: That is not 100% correct.
We actually support aarch64 architecture, and we have the aarch64 tag for that.
If one wants to use it, he can, and should replace latest tag with aarch64 tag.
In the future, we can try to support both architectures in the same docker image using a multi platform image https://docs.docker.com/build/building/multi-platform/

**Comment by rafaeldtinoco** [comment]
@AnaisUrlichs FYI git log messages:

1. keep the "imperative" mood (use fix x,y,z and not fixing x,y,z)
2. use the subject format we have recently adopted if possible

There are tons of git message best practices around there but they are too prolix for me to point you to for simple stuff. So I would recommend your git messages like:

docs(area_being_documented): include tutorial for x
docs(another_area): rewrite paragraph for better understanding
...

If you have any doubts, use "git log" and check the previous commits and they will give you an idea of how to format your git log.

Thank you

Cheers!

**Review by geyslan — APPROVED** [review]
LGTM. Gonna squash and reword commits.

**Comment by AnaisUrlichs** [comment]
Made further changes in accordance with the working solution https://github.com/aquasecurity/tracee/pull/3537

**Pull request merged**